docs/getting-started.md

```diff
@@ -0,0 +1,533 @@
+# Getting Started with Tekton Pipelines
+
+This guide helps you get started with Tekton Pipelines by walking you through a simple `Hello World` tutorial. The tutorial shows you how to:
+
+- Create a `Task`	
+- Create a `Pipeline` containing your `Tasks`	
+- Use a `TaskRun` to instantiate and execute a `Task` outside of a `Pipeline`	
+- Use a `PipelineRun` to instantiate and run a `Pipeline` containing your `Tasks`
+
+## Before you begin
+
+Before you begin this tutorial, make sure you have [installed and configured](install.md)
+the latest release of Tekton on your Kubernetes cluster.
+
+### Install Minikube
+If you would like to complete this tutorial on your local workstation, you will need Minikube.
+Install [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) v1.50 or higher.
+
+### Install Tekton CLI
+This getting started guide uses the Tekton CLI, tkn, so make sure to install that on your local machine.
+Install [Tekton CLI](https://github.com/tektoncd/cli). 
```

Comment:
This might be better if linked to the docs on the website eventually. Unfortunately the docs on the website don't have the ability to jump to certain sections. 
 
For the time being, would recommend changing this link to https://github.com/tektoncd/cli#installing-tkn